xlr/types/src/core.ts

```diff
@@ -13,6 +13,8 @@ export interface Annotations {
   see?: string | Array<string>;
   /** The Typescript comment associated with the type */
   comment?: string;
+  /** The JSDoc `@metatag` string for this type */
+  metatags?: string | Record<string, string>;
```

Comment:
small nitpick but since the name is plural I would assume the type would only be a `Record` and not a singular value. I think just `meta` would be more accurate and also conform to the rest of the existing property names.